Assets/Scripts/Interactables/Collectable.cs

```diff
@@ -1,6 +1,9 @@
+using System;
 using Managers;
+using Player;
 using UnityEngine;
 using Util;
+using Random = UnityEngine.Random;
```

Comment:
`using System` is unused
also if you remove that you can remove the `using Random = UnityEngine.Random`
```suggestion
using Managers;
using Player;
using UnityEngine;
using Util;
```